docs/pages/virtual.md

```diff
@@ -5,18 +5,22 @@ layout: default
-| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: |
-| conda-forge [miniforge] | Installs, runs, and updates packages and their dependencies. Uses `conda`, but with community maintained packages from `conda-forge` channel instead of commercially maintained packages.             | 🟢  |
-| [pipenv]                | Automatically creates and manages a virtualenv for your projects.                                                                                                                                     | 🟠  |
-| [pyenv]                 | Lets you easily switch between multiple versions of Python.                                                                                                                                           | 🟠  |
-| [virtualenv]            | Creates isolated Python environments, and offers more features than venv.                                                                                                                             | 🟠  |
-| [venv]                  | Creates isolated Python environments. Fewer features than other tools, but very widely used and comes built into Python.                                                                              | 🟠  |
-| [anaconda]              | Due to recent [licensing ambiguity][anaconda-problems], we recommend avoiding anaconda and many of the default channels. We recommend installing miniforge and sticking to the `conda-forge` channel. | 🔴  |
+| Name                    | Short description                                                                                                                                                                                                                                            | 🚦  |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :-: |
+| conda-forge [miniforge] | Installs, runs, and updates packages and their dependencies. Uses `conda`, but with community maintained packages from `conda-forge` channel instead of commercially maintained packages.                                                                    | 🟢  |
+| [uv]                    | Fast dependecy (and project) management tool. A drop-in replacement for `venv`. This will probably become our 🟢 tool, but for now we play cautious as there are some open bugs and features. If you're comfortable using `venv` syntax we'd recommend this. | 🟠  |
```

Comment:
```suggestion
| [uv]                    | Fast dependency (and project) management tool. A drop-in replacement for `venv`. This will probably become our 🟢 tool, but for now we are playing cautious as there are some open bugs and features. If you're comfortable using `venv` syntax we'd recommend this. | 🟠  |
```